neurodamus/commands.py

```diff
@@ -71,6 +71,11 @@ def neurodamus(args=None):
         --report-buffer-size=<number> Override the size in MB each rank will allocate for each
                                       report buffer to hold data. When the buffer is full, the
                                       ranks will aggregate data for writing to disk. Default: 8 MB
+        --cell-permute=[0, 1]   Cell permutation [default: 0].
```

Comment:
instead of using numbers, why not use the actual name?
IE: `--cell-permute=[unpermuted, node-adjacency]`

Reply:
yeah